ci_framework/roles/sosreport/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Ensure sos is installed
+  ansible.builtin.yum:
+    name: sos
+    state: present
+  become: yes
+
+- name: Create local "sos" directory
+  ansible.builtin.file:
+    path: "/tmp/sos"
+    state: directory
+
+- name: Provide config file for "sos"
+  ansible.builtin.template:
+    src: sos.conf.j2
+    dest: "/tmp/sos/sos.conf"
+
+- name: Ensure that logs directory is writable
+  ansible.builtin.file:
+    path: "{{ cifmw_sosreport_tmp_dir }}"
+    state: directory
+
+- name: Run "sos collect" on given nodes
```

Comment:
I think sos needs sudo? so a `become: true` may be needed?

Reply:
It doesn't require `become: true` when it runs on remote servers, because we connect to them as `root`.